src/main/java/net/sf/jabref/logic/util/io/FileUtil.java

```diff
@@ -111,46 +110,38 @@
     /**
      * Copies a file.
      *
-     * @param source         File Source file
-     * @param dest           File Destination file
-     * @param deleteIfExists boolean Determines whether the copy goes on even if the file
-     *                       exists.
-     * @return boolean Whether the copy succeeded, or was stopped due to the
-     * file already existing.
+     * @param pathToSourceFile File Source file
+     * @param pathToDestinationFile File Destination file
+     * @param deleteIfExists boolean Determines whether the copy goes on even if the file exists.
+     * @return boolean Whether the copy succeeded, or was stopped due to the file already existing.
      * @throws IOException
      */
-    public static boolean copyFile(File source, File dest, boolean deleteIfExists) throws IOException {
+    public static boolean copyFile(Path pathToSourceFile, Path pathToDestinationFile, boolean deleteIfExists) throws IOException {
         // Check if the file already exists.
-        if (dest.exists() && !deleteIfExists) {
+        if (!Files.exists(pathToSourceFile)) {
             return false;
         }
-        try (BufferedInputStream in = new BufferedInputStream(new FileInputStream(source));
-                BufferedOutputStream out = new BufferedOutputStream(new FileOutputStream(dest))) {
-
-
-            int el;
-            while ((el = in.read()) >= 0) {
-                out.write(el);
-            }
-            out.flush();
+        if (Files.exists(pathToDestinationFile) && !deleteIfExists) {
+            return false;
         }
+        Files.copy(pathToSourceFile, pathToDestinationFile, StandardCopyOption.REPLACE_EXISTING);
```

Comment:
You should  check for null if it was succesful:
` return Files.copy(source, dest, StandardCopyOption.REPLACE_EXISTING) != null;`